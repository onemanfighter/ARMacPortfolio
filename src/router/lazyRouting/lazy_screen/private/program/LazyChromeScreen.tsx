import { lazy } from 'react';
import { LazyProvider } from '../../../component';

const Chrome = lazy(
  () => import('../../../../../screens/private/program/Chrome/Chrome'),
);

const LazyChromeComponent = () => {
  return (
    <LazyProvider>
      <Chrome />
    </LazyProvider>
  );
};

export { LazyChromeComponent };
