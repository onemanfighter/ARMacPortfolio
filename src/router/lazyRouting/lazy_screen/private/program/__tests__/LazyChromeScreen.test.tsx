import { render, screen, waitFor } from '@testing-library/react';
import { LazyChromeComponent } from '../LazyChromeScreen';

describe('LazyChromeComponent', () => {
  it('should render correctly to match snapshot', async () => {
    const { container } = render(<LazyChromeComponent />);

    await waitFor(() => expect(screen.getByText('Chrome')).toBeDefined());
    expect(container).toMatchSnapshot();
  });
});
