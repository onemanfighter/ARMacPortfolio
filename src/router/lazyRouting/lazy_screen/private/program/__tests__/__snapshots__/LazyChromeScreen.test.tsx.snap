// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LazyChromeComponent should render correctly to match snapshot 1`] = `
<div>
  <div>
    <h1>
      Chrome
    </h1>
  </div>
</div>
`;
